### DISCUSSION TITLE
Managing Drap&Drop

### DISCUSSION THREAD

**Discussion by Solido** [btwld/mix#468] [Q&A]
I'm using Mix with Drag&Drop feature and Theme as the top of my app.
Using Drag&Drop create a new Context where the theme is not accessible.
What's the recommanded Mix way to solve that? Thanks

**Comment by tilucasoli** [comment]
Hey there! Thanks for your contribution. I'll look into this issue and do some research to find the best way to solve it. I'll get back to you with a recommended Mix solution soon.

**Comment by Solido** [comment]
This is not handled by Flutter itself, you can use Portal or others tricks to copy the data in both contexts. Let me know if you have any hint else I'll tinker smth on my side as this is now blocking for the app I'm working on with Red Screen of Death :)
